mlrun/config.py

```diff
@@ -691,6 +691,7 @@
         # supported modes: "enabled", "disabled".
         "mode": "disabled"
     },
+    "auth_with_client_id": False,
```

Comment:
can be under `httpdb.authentication` no?

Reply:
I thought about it, but as a matter of fact the `httpdb` part in the config is server-side configuration (with a terrible name), and is mostly used by the server side code. I didn't want to break that convention, though the temptation was there.